Rewards/Classes/Account/AccountService.swift

```diff
@@ -93,7 +94,14 @@ func login(username: String, password: String, provider: AccountProvider) throws
             throw TikiError.error("Account already linked.")
         } else {
             let account = Account(username: username, provider: provider, status: .verified)
-            _accounts.append(account)
+            switch(provider){
+            case .retailer(let retailerEnum):
```

Comment:
it would be better for readability if we just do the switch for accountType, not the whole function.
something like:
```
var accountType: AccountType?
switch(provider){
            case .retailer(let retailerEnum):
            accountType = .retailer(let retailerEnum)
            break;
            case .email(let emailEnum):
            accountType = .email(let emailEnum)
            break;
            }
            ```

Reply:
done